diff --git a/libhb/encsvtav1.c b/libhb/encsvtav1.c
index 217dc0d76..f8ae7d278 100644
--- a/libhb/encsvtav1.c
+++ b/libhb/encsvtav1.c
@@ -187,7 +187,7 @@ int encsvtInit(hb_work_object_t *w, hb_job_t *job)
         }
     }
 
-    if (job->encoder_tune != NULL && strstr(job->encoder_tune, "ms-ssim") != NULL)
+    if (job->encoder_tune != NULL && strstr(job->encoder_tune, "grain") != NULL)
     {
         param->tune = 4;
     }

diff --git a/libhb/handbrake/av1_common.h b/libhb/handbrake/av1_common.h
index 50ea96fd0..d1fc31ceb 100644
--- a/libhb/handbrake/av1_common.h
+++ b/libhb/handbrake/av1_common.h
@@ -31,7 +31,7 @@ static const char * const hb_av1_svt_preset_names[] =
 
 static const char * const hb_av1_svt_tune_names[] =
 {
-    "vq", "psnr", "ssim", "iq", "ms-ssim", "fastdecode", NULL
+    "vq", "psnr", "ssim", "iq", "grain", "fastdecode", NULL
 };
 
 static const char * const hb_av1_svt_profile_names[] =
