diff --git a/libhb/handbrake/av1_common.h b/libhb/handbrake/av1_common.h
index 50ea96fd0..677e6902e 100644
--- a/libhb/handbrake/av1_common.h
+++ b/libhb/handbrake/av1_common.h
@@ -26,7 +26,7 @@ static const int          hb_av1_level_values[] = {
 
 static const char * const hb_av1_svt_preset_names[] =
 {
-    "10", "9", "8", "7", "6", "5", "4", "3", "2", "1", "0", "-1", NULL
+    "8-faster", "6-fast", "5-medium", "4-slow", "2-slower", NULL
 };
 
 static const char * const hb_av1_svt_tune_names[] =
