diff --git a/contrib/svt-av1/module.defs b/contrib/svt-av1/module.defs
index a63058238..2a595fd02 100644
--- a/contrib/svt-av1/module.defs
+++ b/contrib/svt-av1/module.defs
@@ -1,9 +1,8 @@
 $(eval $(call import.MODULE.defs,SVT-AV1,svt-av1))
 $(eval $(call import.CONTRIB.defs,SVT-AV1))
 
-SVT-AV1.FETCH.url     = https://github.com/HandBrake/HandBrake-contribs/releases/download/contribs2/SVT-AV1-v3.1.2.tar.gz
-SVT-AV1.FETCH.url    += https://gitlab.com/AOMediaCodec/SVT-AV1/-/archive/v3.1.2/SVT-AV1-v3.1.2.tar.gz
-SVT-AV1.FETCH.sha256  = d0d73bfea42fdcc1222272bf2b0e2319e9df5574721298090c3d28315586ecb1
+SVT-AV1.FETCH.url     = https://github.com/kentooo/HandBrake-SVT-AV1-PSYEX/releases/download/contribs/svt-av1-hdr.tar.gz
+SVT-AV1.FETCH.sha256  = e01281a2a25242dcd0a14cb0963ce979e8138b1a847ca9557a939b13a4dcea57
 
 SVT-AV1.GCC.args.c_std =
 
@@ -26,11 +25,9 @@ else
     endif
 endif
 
-ifeq (on,$(GCC.lto))
-    SVT-AV1.CONFIGURE.extra += -DSVT_AV1_LTO=ON
-else ifeq (off,$(GCC.lto))
-    SVT-AV1.CONFIGURE.extra += -DSVT_AV1_LTO=OFF
-endif
+SVT-AV1.CONFIGURE.extra += -DSVT_AV1_LTO=ON
+SVT-AV1.CONFIGURE.extra += -DCMAKE_CXX_FLAGS_RELEASE="-flto -DNDEBUG -pipe -O3"
+SVT-AV1.CONFIGURE.extra += -DCMAKE_C_FLAGS_RELEASE="-flto -DNDEBUG -pipe -O3"
 
 ifeq (native,$(GCC.cpu))
     SVT-AV1.CONFIGURE.extra += -DNATIVE=ON
